drivers/gles3/shaders/copy.glsl

```diff
@@ -94,6 +99,8 @@ uniform samplerCube source_cube; //texunit:0
 uniform sampler3D source_3d; //texunit:0
 #elif defined(USE_TEXTURE2DARRAY)
 uniform sampler2DArray source_2d_array; //texunit:0
+#elif defined(USE_EXTERNAL_SAMPLER)
+uniform samplerExternalOES source;
 #else
```

Comment:
One of the remarks someone made on the original PR was that using textureLOD further down to access our texture doesn't work on some android GPUs, we may want to change the code to texture. I can't test this because textureLod is working on my device.

Reply:
Done!